<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HIntertrialIntervalSettingsForm</class>
 <widget class="QWidget" name="HIntertrialIntervalSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Intertrial interval</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbNoIntertrialInterval">
        <property name="text">
         <string>No intertrial interval</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUseIntertrialInterval">
        <property name="toolTip">
         <string>Insert a gap, with optional stimulus, between consecutive trials.</string>
        </property>
        <property name="text">
         <string>Use an intertrial interval</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbIntervalType">
        <property name="title">
         <string>Interval type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="rbFixedTimeInterval">
             <property name="text">
              <string>Fixed time interval</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leFixedTimeMS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="rbExperimenterControlled">
           <property name="text">
            <string>Experimenter controlled interval length</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbStimulus">
        <property name="title">
         <string>Intertrial stimulus</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="rbStimulus">
             <property name="toolTip">
              <string>No gap between trials.</string>
             </property>
             <property name="text">
              <string>Use a stimulus as an attention getter</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSound">
             <property name="text">
              <string>Use a sound-only stimulus</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbBackground">
             <property name="text">
              <string>No stimulus, background color only</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbEditStimulus">
           <property name="text">
            <string>Edit Stimulus</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>gbIntervalType</zorder>
     <zorder>rbUseIntertrialInterval</zorder>
     <zorder>rbNoIntertrialInterval</zorder>
     <zorder>gbStimulus</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbUseIntertrialInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbIntervalType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFixedTimeInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>leFixedTimeMS</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbStimulus</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbEditStimulus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSound</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbEditStimulus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbEditStimulus</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUseIntertrialInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbStimulus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
